content/gettingstarted/index.adoc

```diff
@@ -22,123 +22,125 @@ Kiali will support the latest Istio version (1.1 at the moment) and the developm
 
-icon:bullhorn[size=1x]{nbsp} This script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
+To install the Kiali Operator (and, by default, the Kiali CR), run the install script via this command:
 
 [source,bash]
 ----
-bash <(curl -L http://git.io/getLatestKialiKubernetes)
+bash <(curl -L https://git.io/getLatestKialiOperator)
 ----
 
-By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKialiKubernetes[the comments at the top of the install script].
+You will be prompted for the authentication strategy you want to use. Choose "login" if you want to define a username and password that users will be required to access Kiali. Choose "anonymous" to allow full access to Kiali without requiring any credentials. Choose "openshift" to use the OpenShift OAuth login which controls access based on the individual's RBAC roles in OpenShift (this is only a feature available when deployed on OpenShift).
 
-Once this is done, Kiali will be deployed and running in Kubernetes with an Ingress set up.
+If the authentication strategy of "login" is selected, a secret containing Kiali login credentials is required to be deployed along with Kiali. In this case, the install script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The install script will store those credentials in a secret that is deployed in the same namespace where Kiali is installed.
 
-If you deployed in minikube, you can go directly to the Kiali UI at the URL `https://[minikube-ip]/kiali/console` where `[minikube-ip]` is what is reported when you run the command: `minikube ip`.
+icon:bullhorn[size=1x]{nbsp} If you told the install script to not create a Kiali CR (and thus not have Kiali installed yet) via the OPERATOR_INSTALL_KIALI=false environment variable, you are responsible for creating this secret if you wish to install Kiali with the authentication strategy of "login". A secret is not required if your authentication strategy is not "login".
+
+Once the Kiali Operator has installed Kiali and the Kiali pod has successfully started, you can begin to access Kiali.
+
+If on OpenShift, you can easily find the URL to access Kiali by going to the OpenShift console and navigating to the `istio-system` project's Overview page:
+
+[#img-openshift]
+.OpenShift console
+image::os-console.png[OpenShift console]
+{nbsp} +
+
+In this case, the Kiali URL is `http://kiali-istio-system.192.168.64.13.nip.io`. In your case this will be a different IP.
+
+You can also use the `oc` (or `kubectl` if on Kubernetes) command to determine the base URL:
```

Comment:
detecting something similar on k8s?

Reply:
I added a sentence here to say go through the k8s Ingress. I don't have a running k8s instance, so I can't give exact commands/details.